novawallet/Modules/PayCard/PayCardInteractor.swift

```diff
@@ -3,46 +3,82 @@ import Foundation
 final class PayCardInteractor {
     weak var presenter: PayCardInteractorOutputProtocol?
 
-    let chainRegistry: ChainRegistryProtocol
+    let payCardModelFactory: PayCardHookFactoryProtocol
```

Comment:
It seems that `payCardModelFactory` is a bit misleading. `PayCardModel` is created below without using any model factory, and `payCardModelFactory` is actually a hooks factory. `payCardHookFactory` might be a better fit here.